src/System.Runtime.InteropServices.WindowsRuntime/tests/System/Runtime/InteropServices/WindowsRuntime/WindowsRuntimeMarshalTests.cs

```diff
@@ -281,25 +281,25 @@ public void StringToHString_PtrToHString_ReturnsExpected(string s)
             }
         }
 
-        [ConditionalFact(nameof(PlatformDetection) + "." + nameof(PlatformDetection.IsWinRTSupported))]
+        [ConditionalFact("System.PlatformDetection, CoreFx.Private.TestUtilities!" + nameof(PlatformDetection.IsWinRTSupported))]
```

Comment:
I suppose we could make ConditionalXXX treat 
`[ConditionalFact(nameof(PlatformDetection.IsWinRTSupported))]`
as shorthand for 
`[ConditionalFact("System.PlatformDetection, CoreFx.Private.TestUtilities!" + nameof(PlatformDetection.ZZZ))]`
ie special case platformdetection.

But everyone will copy paste this anyway so fine for now.